A theme change + some more current language (#2)
rolloff/wrc-east-bay: faq.md

# FAQ
## I have a friend who wants to join. How can I get them added to Slack? 
Send them this form: [https://tinyurl.com/wrc-jointheteam](https://tinyurl.com/wrc-jointheteam). Our onboarding committee will commit to adding in new folks on a monthly basis.

> ## Will this be competing with other East Bay run clubs? 

Reviewer: Let's move this towards the end

> There is not (that we know of!) an active women’s run club in the East Bay that has local roots, spans many running disciplines, and is free or low cost to join. In that sense, we hope this will be something new that will serve runners who we may not have met yet - runners craving similar paced teammates or looking for a women-centered space without a high annual price tag. For runners already plugged into an existing club, we hope that our group can be a value-add to the broader running community ecosystem - we are a “Yes and” if you will. Hopefully we can also have some joint runs with other East Bay running groups.

Reviewer: I'm trying to remove the comparisons to other groups:
```suggestion
We are a women’s running club in the East Bay that has local roots, spans many running disciplines, and is free to join. We hope this will be something new that will serve runners who we may not have met yet - runners craving similar-paced teammates or looking for a women-centered space. For runners already plugged into an existing club, we hope that our group can be a value-add to the broader running community ecosystem - we are a “Yes and” if you will. We have had and would love to continue collaborations with other East Bay running groups.
```

## Is this a road specific run group? 

> We plan on scheduling road, track, and trail group runs. And there may be some members training for triathlons!

Reviewer: ```suggestion
We have road, track, and trail group runs. And there are even some members training for triathlons!
```

## How do I find out about run meet ups?
Join us on our Slack space for the most up to date details. We also post about most of our runs on our [instagram](https://www.instagram.com/wrceastbay).

## How can I help? 

> Inviting a friend to WRC, organizing ad-hoc runs in #run-meets in Slack while we get off the ground, and starting conversations in other WRC Slack channels is a great start. Beyond that, areas that will likely need help:

Reviewer: ```suggestion
Inviting a friend to WRC, organizing ad-hoc runs in #run-meets in Slack, and starting conversations in other WRC Slack channels is a great start. Beyond that, areas that will likely need help:
```

> - Creating a social media presence or maybe a website or listserv

Reviewer: This feels outdated, delete it?

- Onboarding new members (e.g. adding to Slack and making sure new folks have friendly faces to welcome them at their first run)

> - Planning monthly programming in 2023

Reviewer: Also, leading planned runs?

- Organizing Slack channels, adding more fun emojis, starting conversation, and norming
- Outreach
- Writing community guidelines / code of conduct for slack
- Leadership / Strategy 
If you are interested in any of these please let us know on Slack or email. 

## I have feedback and questions. Who do I ask? 

> Please share feedback, ideas, and questions in whatever medium is most comfortable to you. As a starting point, feel free to post in the general channel on WRC Slack or via Slack DM if you prefer a more private conversation.

Reviewer: Should this say who to DM? Becca, Ali, Nicole?
